addons/l10n_latam_document/models/account_chart_template.py

```diff
@@ -0,0 +1,23 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import models, api, fields, _
+
+
+class AccountChartTemplate(models.Model):
+
+    _inherit = 'account.chart.template'
+
```

Comment:
can not we add the global rounding in one of the methods of installing the chart template?

Reply:
Ok, added in the commit